### PR TITLE
Hx711

### PR DESCRIPTION
**Pull request by JSiuda1**

### CODE REVIEW

File: components/cmd/valves_cmd.c [hunk 1]
```diff
@@ -71,22 +71,6 @@ static int cmd_valve_time_open(int argc, char **argv) {
     return 0;
 }
 
-static int cmd_valve_read_data(int argc, char **argv) {
```
Comment (kgliwinski) [inline components/cmd/valves_cmd.c:74]: Why is this deleted?

**Review by kgliwinski — COMMENTED** [review]
Don't like the HX lib, could be optimised but as long as it works its ok.

Please answer my comment before deciding to close 🙂

File: components/cmd/valves_cmd.c (hunk 1 above)
Comment (JSiuda1) [reply components/cmd/valves_cmd.c:74]: Has been moved to ```common_cmd.c``` file, function ```cmd_read_data```.

Comment (JSiuda1) [reply components/cmd/valves_cmd.c:74]: Has been moved to the ```common_cmd.h``` file, function ```read_data```.

**Review by kgliwinski — APPROVED** [review]
